pkgs/development/libraries/opencv/3.x.nix

```diff
@@ -15,6 +15,7 @@
 , enableFfmpeg    ? false, ffmpeg
 , enableGStreamer ? false, gst_all_1
 , enableEigen     ? false, eigen
+, enableOpenblas  ? false, openblas
```

Comment:
@viric @mdaiter what criterion did you choose, which features to enable in opencv?